hack/builder/Dockerfile

```diff
@@ -96,4 +98,6 @@ COPY rsyncd.conf /etc/rsyncd.conf
 
 COPY entrypoint.sh /entrypoint.sh
 
+COPY create_bazel_cache_rcs.sh /create_bazel_cache_rcs.sh
```

Comment:
where is this file coming from? i don't see it generated or in the src tree.

Reply:
It was originally downloaded in the build.sh from test-infra bootstrap. I changed it to copy the relevant parts as an own script into the builder directory.